### ISSUE TITLE
Make it possible to store only username for auth repository

### ISSUE THREAD

**Issue by rominf** [sdispater/poetry#478]
- [x] I have searched the [issues](https://github.com/sdispater/poetry/issues) of this repo and believe that this is not a duplicate.

## Issue

I want to store username for repository, but don't want to store a password because of security issues. If I manually modify `~/.config/pypoetry/auth.toml` file to look like this:
```
[http-basic]                                                                                                                      
[http-basic.test]                                                                                                                 
username = "Roman.Inflianskas"
```
I get this on my attempts to publish the package (I expect that `poetry` will ask me missing password):
```
$ poetry publish -r test

Publishing ordered_set-stubs (0.1.0) to test
                                    
[NonExistentKey]     
'Key "password" does not exist.'  
                                    
publish [-r|--repository REPOSITORY] [-u|--username USERNAME] [-p|--password PASSWORD] [--build]
```

**Comment by sdispater** [comment]
It's not possible at the moment but I think this would be a nice improvement.

**Comment by sdispater** [comment]
Available in the latest `1.0.0` beta release.
